src/ops/kernels/element_binary_kernels.cu

```diff
@@ -489,6 +539,8 @@ void backward_kernel(ElementBinaryMeta const *m,
                                  in2_grad_ptr));
       }
     }
+  } else if (m->op_type == OP_EW_MIN || m->op_type == OP_EW_MAX) {
+    // TODO: Implement backward pass
```

Comment:
@jiazhihao, do you have some/any pointers about implementing the backward pass for elementwise max and min using CUDNN? I wasn't able to figure it out.